install_config/install/prerequisites.adoc

```diff
@@ -197,14 +197,68 @@ The following must be set up in your environment before OpenShift can be
 installed.
 
 [[prereq-dns]]
-
 === DNS
 
+A fully functional DNS environment is an absolute requirement for
+{product-title} to work correctly. It is not enough to simply add entries for
+hosts in the *_/etc/hosts_* file, because the DNS resolver libraries do not
+query that file. Also, key components of OSE v3 run themselves inside of
+containers. By default, these containers receive their *_/etc/resolv.conf_* DNS
+configuration file from their host. {product-title} then inserts one DNS value
+into the pods (above the node's nameservers values). That value is defined in
+the *_/etc/origin/node/node-config.yaml_* by the `*dnsIP*` parameter. By default
+this will be the IP address of the host node since *dnsmasq* is configured.
```

Comment:
By default this IP will be set to the IP address of the host node since *dnsmasq* is being use on the host. If `*dnsIP*` parameter left out of the config then the value will be defaulted to the kubernetes service IP being inserted as the first nameserver in the pods resolv.conf file.